be/src/vec/exec/format/parquet/parquet_bloom.h

```diff
@@ -0,0 +1,356 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#pragma once
+
+#include <io/fs/file_reader.h>
+#include <parquet/hasher.h>
+#include <parquet/types.h>
+
+namespace doris::vectorized {
+class BloomFilter {
+public:
+    // Maximum Bloom filter size, it sets to HDFS default block size 128MB
+    // This value will be reconsidered when implementing Bloom filter producer.
+    static constexpr uint32_t kMaximumBloomFilterBytes = 128 * 1024 * 1024;
+
+    /// Determine whether an element exist in set or not.
+    ///
+    /// @param hash the element to contain.
+    /// @return false if value is definitely not in set, and true means PROBABLY
+    /// in set.
+    virtual bool FindHash(uint64_t hash) const = 0;
```

Comment:
```suggestion
    virtual bool find_hash(uint64_t hash) const = 0;
```
Same as other methods' name

Reply:
fixed